mitodl/mitxpro
# Added validation for legal address fields that need it

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #430 

#### What's this PR do?
Adds validators to the `state_or_territory` and `country` fields which must both be valid ISO-3166 values so that the shell and django-admin will validate inputs

#### How should this be manually tested?
Go to django-admin for a user and try to save garbage values (e.g. insert a number). It should fail with an error message.
